packer/scripts/modules/Invoke-CreateVMSwitch.ps1

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
should this be at the same directory level as the build-vm.ps1 script? this will need to be ran before it.